doc/Language/5to6-nutshell.pod6

```diff
@@ -890,6 +890,17 @@ for @array -> $first, $second {
 }
 =end code
 
+For instances in which the number of elements iterated over isn't a multiple
```

Comment:
I am almost sure "For cases" is more natural than "For instances".
Maybe "For cases when the number of..."?